### PR TITLE
Update hide column about IoT packages in csv files

### PR DESCRIPTION
**Pull request by v-xuto**
**Updates：**

1.  Update hide column about IoT packages in CSV files by email CSV Updates.
2.  Update the display name of the package  `Azure.Iot.DeviceUpdate` and  `Azure.Iot.ModelsRepository`  in four languages.

**Questions:**
In this PR, I changed hide to true about package `azure-iot-deviceupdate` in java according to the email. But, I wonder if the package `azure-iot-deviceupdate` in java needs to be hide? This package in the other three languages is not hidden.

@jongio for notification.

### CODE REVIEW

**Comment by jongio** [comment]
I added elena  @tzhanl 

LGTM

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -443,25 +443,25 @@
...
-"iot-deps","com.microsoft.azure.sdk.iot","0.11.2","","IoT Deps","IoT","NA","NA","NA","client","false","","","","","",""
-"iot-deps-preview","com.microsoft.azure.sdk.iot","1.1.0","","IoT Deps","IoT","NA","NA","NA","client","false","","","","true","",""
-"iot-deps-serializer","com.microsoft.azure","0.3.3","","IoT Deps Serializer","IoT","NA","NA","NA","client","false","","","","true","",""
-"iot-deps-serializer","com.microsoft.azure.sdk.iot.deps","0.7.6","","IoT Deps Serializer","IoT","NA","NA","NA","client","false","","","","","",""
-"iot-device-client","com.microsoft.azure","1.1.24","","IoT Device Client","IoT Plug and Play","NA","NA","NA","client","false","","","","true","",""
-"iot-device-client","com.microsoft.azure.sdk.iot","1.29.4","","IoT Device Client","IoT Plug and Play","NA","NA","NA","client","false","","","","","",""
-"iot-device-client-preview","com.microsoft.azure.sdk.iot","1.2.0","","IoT Device Client","IoT Plug and Play","NA","NA","NA","client","false","","","","true","",""
+"iot-deps","com.microsoft.azure.sdk.iot","0.11.2","","IoT Deps","IoT","NA","NA","NA","client","false","","","","false","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:446]: @elraikhm Did we intend to unhide these packages?

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -21,7 +21,7 @@
 "azure-core-serializer-json-gson","com.azure","1.1.1","","Core Serializer GSON JSON","Core","core","","","client","true","","09/04/2020","active","","",""
 "azure-core-serializer-json-jackson","com.azure","1.2.2","","Core Serializer Jackson JSON","Core","core","","","client","true","","09/04/2020","active","","",""
 "azure-cosmos","com.azure","4.14.0","4.15.0-beta.1","Cosmos DB","Cosmos DB","cosmos","","","client","true","","06/10/2020","active","","com.microsoft.azure/azure-cosmos",""
-"azure-iot-deviceupdate","com.azure","","1.0.0-beta.1","Device Update","IoT","deviceupdate","","","client","true","","","","","",""
+"azure-iot-deviceupdate","com.azure","","1.0.0-beta.1","IoT Device Update","IoT Device Update","deviceupdate","","","client","true","","","","true","",""
```
Comment (elraikhm) [inline _data/releases/latest/java-packages.csv:24]: Let's not hide it for now

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (elraikhm) [reply _data/releases/latest/java-packages.csv:446]: IoT team say this one is active

File: _data/releases/latest/java-packages.csv [hunk 3]
```diff
@@ -443,25 +443,25 @@
...
-"iot-deps-preview","com.microsoft.azure.sdk.iot","1.1.0","","IoT Deps","IoT","NA","NA","NA","client","false","","","","true","",""
-"iot-deps-serializer","com.microsoft.azure","0.3.3","","IoT Deps Serializer","IoT","NA","NA","NA","client","false","","","","true","",""
-"iot-deps-serializer","com.microsoft.azure.sdk.iot.deps","0.7.6","","IoT Deps Serializer","IoT","NA","NA","NA","client","false","","","","","",""
-"iot-device-client","com.microsoft.azure","1.1.24","","IoT Device Client","IoT Plug and Play","NA","NA","NA","client","false","","","","true","",""
-"iot-device-client","com.microsoft.azure.sdk.iot","1.29.4","","IoT Device Client","IoT Plug and Play","NA","NA","NA","client","false","","","","","",""
-"iot-device-client-preview","com.microsoft.azure.sdk.iot","1.2.0","","IoT Device Client","IoT Plug and Play","NA","NA","NA","client","false","","","","true","",""
+"iot-deps","com.microsoft.azure.sdk.iot","0.11.2","","IoT Deps","IoT","NA","NA","NA","client","false","","","","false","",""
+"iot-deps-preview","com.microsoft.azure.sdk.iot","1.1.0","","IoT Deps","IoT","NA","NA","NA","client","false","","","","false","",""
```
Comment (elraikhm) [inline _data/releases/latest/java-packages.csv:447]: Following packages should be hidden (set Hide to true)
iot-deps-preview | com.microsoft.azure.sdk.iot | 1.1.0
iot-deps-serializer | com.microsoft.azure | 0.3.3
iot-deps-serializer | com.microsoft.azure.sdk.iot.deps | 0.7.6
iot-device-client | com.microsoft.azure | 1.1.24
iot-device-client-preview | com.microsoft.azure.sdk.iot | 1.2.0

File: _data/releases/latest/java-packages.csv [hunk 4]
```diff
@@ -443,25 +443,25 @@
...
-"iot-service-client","com.microsoft.azure","1.2.18","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","true","",""
+"iothub-react_2.12.0-RC1","com.microsoft.azure.iot","0.7.0","","IoT Hub React","IoT Hub","NA","NA","NA","client","false","","","","true","",""
+"iot-service-client","com.microsoft.azure","1.2.18","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","false","",""
 "iot-service-client","com.microsoft.azure.sdk.iot","1.28.0","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","","",""
-"iot-service-client-preview","com.microsoft.azure.sdk.iot","1.2.0","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","true","",""
+"iot-service-client-preview","com.microsoft.azure.sdk.iot","1.2.0","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","false","",""
```
Comment (elraikhm) [inline _data/releases/latest/java-packages.csv:464]: Hide = true for iot-service-client-preview | com.microsoft.azure.sdk.iot

File: _data/releases/latest/java-packages.csv [hunk 5]
```diff
@@ -443,25 +443,25 @@
...
 "iothub-java-service-client-dev","com.microsoft.azure.iothub-java-client","1.0.2","","IoT Hub Java Service Client","IoT Hub","NA","NA","NA","client","false","","","","true","",""
 "iothub-react_2.11","com.microsoft.azure.iot","0.9.0","","IoT Hub React","IoT Hub","NA","NA","NA","client","false","","","","true","",""
 "iothub-react_2.12","com.microsoft.azure.iot","0.9.0","","IoT Hub React","IoT Hub","NA","NA","NA","client","false","","","","true","",""
-"iothub-react_2.12.0-RC1","com.microsoft.azure.iot","0.7.0","","IoT Hub React","IoT Hub","NA","NA","NA","client","false","","","","","",""
-"iot-service-client","com.microsoft.azure","1.2.18","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","true","",""
+"iothub-react_2.12.0-RC1","com.microsoft.azure.iot","0.7.0","","IoT Hub React","IoT Hub","NA","NA","NA","client","false","","","","true","",""
+"iot-service-client","com.microsoft.azure","1.2.18","","IoT Service Client","IoT","NA","https://docs.microsoft.com/java/api/overview/azure/iot","NA","client","false","","","","false","",""
```
Comment (elraikhm) [inline _data/releases/latest/java-packages.csv:462]: iot-service-client | com.microsoft.azure | 1.2.18 should be hidden

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (elraikhm) [reply _data/releases/latest/java-packages.csv:446]: I mean the iot-deps | com.microsoft.azure.sdk.iot

File: _data/releases/latest/python-packages.csv [hunk 6]
```diff
@@ -369,4 +369,4 @@
 "azure-mgmt-storageimportexport","0.1.0","","Resource Management - Storage Import Export","Storage","storage","NA","","mgmt","false","","","preview","","",""
 "azure-mgmt-trafficmanager","0.51.0","","Resource Management - Traffic Manager","Traffic Manager","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-trafficmanager_0.51.0/azure-mgmt-trafficmanager","https://docs.microsoft.com/python/api/overview/azure/traffic-manager","","mgmt","false","","","preview","","",""
 "azure-mgmt-vmwarecloudsimple","0.2.0","","Resource Management - VM Ware Cloud Simple","VMWare by CloudSimple","compute","NA","","mgmt","false","","","preview","","",""
-"azure-servicemanagement-legacy","0.20.7","","Service Management - Legacy","Core","core","NA","NA","mgmt","false","","","community","","",""
+"azure-servicemanagement-legacy","0.20.7","","Service Management - Legacy","Core","core","NA","NA","mgmt","false","","","community","","",""
```
Comment (elraikhm) [inline _data/releases/latest/python-packages.csv:372]: Support should be "Active" for
iothub-client | 1.1.2.0
iothub-service-client | 1.1.0.0

**Review by elraikhm — CHANGES_REQUESTED** [review]

File: _data/releases/latest/java-packages.csv (hunk 2 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: @elraikhm - I have updated based on the comments.

File: _data/releases/latest/java-packages.csv (hunk 3 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: @elraikhm - I have updated based on the comments.

File: _data/releases/latest/java-packages.csv (hunk 4 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: @elraikhm - I have updated based on the comments.

File: _data/releases/latest/java-packages.csv (hunk 5 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: @elraikhm - I have updated based on the comments.

File: _data/releases/latest/python-packages.csv (hunk 6 above)
Comment (v-xuto) [reply _data/releases/latest/python-packages.csv:372]: @elraikhm - I have updated based on the comments.

**Comment by v-xuto** [comment]
@elraikhm, @weshaggard -  I have updated according to your comments. If there is no problem, please help merge it.

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
@elhorton I'm going ahead and merging this PR as it looks reasonable to me and these PR's are hard to keep updated. If there are other changes we can file another PR.

**Pull request merged**